src/doc/reference/src/visibility-and-privacy.md

```diff
@@ -40,7 +40,10 @@ With the notion of an item being either public or private, Rust allows item
 accesses in two cases:
 
 1. If an item is public, then it can be used externally through any of its
-   public ancestors.
+   accessible ancestors (either by anyone that has an access to an ancestor
+   in the case of a chain of public ancestors,
+   or by the siblings of a private ancestor, in the case of a chain of public
+   ancestors terminating into a private ancestor).
```

Comment:
I puzzled over this paragraph for a good minute and still don't understand what it wants to convey to the reader.

The point of these paragraphs 1 and 2 is that a private item can be used everywhere inside of its module, but not outside it, even through reexports, and `pub` item may be used everywhere where we can name it either directly or through reexports/aliases.
This rule applies to modules in the same way as to everything else, and the facts about "chains of public ancestors" and "siblings of a private ancestor" are some obscure consequences of this general idea (which are also incorrect in presence of reexports).